### ISSUE TITLE
Analytics Implementation or QA Support for Digitization Team-Form 26-4555

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#53146]
# Google Analytics Implementation or QA Request

## What this form is for

<details>
<summary>More Information</summary>
<div>
    <br>
<p>Use this template to request Google Analytics implementation or QA for your product. Please fill out all brackets.</p>
    
> _Please see the following information for implementation timeline:_
> New Teams: Implementation will usually be completed in 4 sprints
> 
> - Sprint 1: Discovery/Orientation
> - Sprint 2: FE Implementation
> - Sprint 3: Tagging Implementation
> - Sprint 4: QA
> 
> All other teams: Implementation will usually be completed in 2 sprints
> 
> - Sprint 1: Discovery and FE Implementation
> - Sprint 2: Tagging Implementation and QA

</div>
</details>

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [x] Google Analytics implementation
     - [ ] QA

- **Who is this request for?**
Forms-digitize
    - [Insert team name(s), for individual's names please use the `/support` tracker on the #vfs-platform-support channel and link this ticket.]
    
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
    - [Staging URL](https://staging.va.gov/housing-assistance/disability-housing-grants/adapted-housing/introduction)

    
    
- **When is the launch date?**
    - Around March 16th, 2023

-  - [Link to Interactions](https://docs.google.com/spreadsheets/d/1krS0RLSjrT6VGmYk1sNF3GlypOSkXh9kJGKoCqUaFy4/edit#gid=0)

    
    - [If this is a funnel, insert each step's URL - pages visible depending on the applicant's answers should be marked accordingly]
    
    - [Insert other interactions - please review the [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) and [GA Data Dictionary](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Google-Analytics-Data-Dictionary.1810464912.html)  to make sure these interactions aren't already tracked]

### Optional
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - [Insert test user]

- **Any additional comments**

    - [Insert comments]

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by michelle-dooley** [comment]
@tbaker1026 could you please provide some more details to this request.  For example the urls of the page you are inquiring about and the events you want to track on those pages.

**Comment by tbaker1026** [comment]
@michelle-dooley I think I opened this one too early, still waiting on the devs for URLs etc. Should I close it and open a new one when ready to fill it out?

**Comment by tbaker1026** [comment]
@michelle-dooley - I believe this is ready with the links and staging URL, let me know if you have any questions

**Comment by michelle-dooley** [comment]
@tbaker1026 - thanks.  I will add this to our backlog to pick up in our next sprint.

**Comment by michelle-dooley** [comment]
@tbaker1026 I apologize but when I said next sprint I forgot I was going to be out for most of it and I am the only one who does these requests so I am going to have to move this to the following sprint starting 3/30

**Comment by tbaker1026** [comment]
@michelle-dooley thank you for the update, will this form also show up on the DOMO dashboard where the other existing online forms are displayed or does that come later?

**Comment by michelle-dooley** [comment]
Getting it added to the [Forms Dashboard](https://va-gov.domo.com/page/447193050) is a separate request.  Here is the [dashboard request ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=joanneesteban&labels=analytics-insights%2C+analytics-request%2C+kpi-dashboard&template=analytics-request-kpi-dashboard.md&title=Analytics+KPI+dashboard+request+for+%5BTeam%5D)



![image](https://user-images.githubusercontent.com/93597659/225419655-793adb1d-a603-4a5d-ab9d-d2eef1f67c65.png)

**Comment by michelle-dooley** [comment]
Closing this ticket as this work was addressed as part of the domo dashboard set up ticket that was just completed.
